Src/BootCamp.Chapter/Program.cs

```diff
@@ -1,16 +1,56 @@
 ﻿using System;
-using System.Collections.Generic;
-using System.Globalization;
 using System.IO;
-using System.Linq;
 using System.Text;
 
 namespace BootCamp.Chapter
 {
-    class Program
+    internal class Program
     {
-        static void Main(string[] args)
+        private static void Main(string[] args)
         {
+            var contents = FileCleaner.Clean(@"Input\Balances.corrupted", @"Input\Balances.clean");
+
+            FindStaticalData(contents); 
+        }
+
+       
+
+
+        private static void FindStaticalData(string[] contents)
+        {
+            // Print each of the statistical output using Text Table with padding 3:
+
+            // - FindHighestBalanceEver
+
+            Console.WriteLine("The richest person ever was : ");
+            var answer = BalanceStats.FindHighestBalanceEver(contents);
+            var names = BalanceParser.ConvertToStringBuilder(answer);
+            var output = $"{names.ToString()} had the most money ever. ¤{answer[0].HighestBalance()}.";
+            Console.WriteLine(TextTable.Build(output, 3));
+
+            //// - FindPersonWithBiggestLoss
+
+            //Console.WriteLine("The person with the biggest loss was  : ");
+            //answer = BalanceStats.FindPersonWithBiggestLoss(contents);
+            //Console.WriteLine(TextTable.Build(answer, 3));
+
+            //// - FindRichestPerson
+
+            //Console.WriteLine("The richest person at this moment is : ");
+            //answer = BalanceStats.FindRichestPerson(contents);
+            //Console.WriteLine(TextTable.Build(answer, 3));
+
+            //// - FindMostPoorPerson
+
+            Console.WriteLine("The poorest person at this moment is : ");
+            answer = BalanceStats.FindMostPoorPerson(contents);
+            names = BalanceParser.ConvertToStringBuilder(answer);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this useless assignment to local variable 'names'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5053435)